namespace cpp entity
namespace py ccentity.welfare
namespace as3 com.dreamfire.sanguo.data.definition.entity

////////////////////////////////////////////////
// 签到配置数值
enum CheckinsConfigType {
  GENERAL_CHECKINS_LIST = 5,
  VIP_CHECKINS_LIST = 5,
  ONLINE_LIST = 4,
}
